woof/star-catcher.html

```diff
@@ -19,27 +19,47 @@
 	</div>
 	<div class="block">
 
-		<p>3) When you click on your shape, have it disappear and reappear in a random location with a random color and size. Bonus: try adding a sound effect! </p>
-		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=kozetic/">Play this demo in full screen.</a>
+		<p>3) When you click on your shape, make it disappear then reappear at a random spot. Bonus: make it play a sound!</p>
```

Comment:
Given that the shape is hiding and then showing immediately, it's really an extra unnecessary step. We could just make it move to a random spot on click.